### PR TITLE
DAH-1313 add unit tests

### PR DESCRIPTION
**Pull request by kramduckner**
[DAH-1313](https://sfgovdt.jira.com/browse/DAH-1313)

### CODE REVIEW

**Comment by christine-sfg** [comment]
As part of this pr, and anytime we're increasing test coverage, can we update the thresholds in [jest.config.js](https://github.com/SFDigitalServices/sf-dahlia-web/blob/3e9b68d4a307bc45546ed25cac183e0ef7eeb729/jest.config.js#L19-L19)? That way we can maintain the new test coverage going forward.

File: app/javascript/__tests__/modules/listings/HabitatForHumanity.test.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import renderer from "react-test-renderer"
+import { getHabitatContent } from "../../../modules/listings/HabitatForHumanity"
+import { habitatListing } from "../../data/RailsSaleListing/listing-sale-habitat"
+
+describe("ListingDetailsHabitat", () => {
+  it("does not display when not habitat listing", () => {
```
[the remark this answers is not in the document]
Comment (kramduckner) [reply app/javascript/__tests__/modules/listings/HabitatForHumanity.test.tsx]: That's actually a copy and paste mistake 😬. It should just be "renders HabitatForHumanity." I went through all of the changes and double checked the wording and made some updates, including the other comment you had! Thanks for catching!!

**Comment by christine-sfg** [comment]
latest changes look good. just waiting on the jest config update. lmk if you want to pair on that @kramduckner

**Comment by kramduckner** [comment]
> latest changes look good. just waiting on the jest config update. lmk if you want to pair on that @kramduckner

Thanks for the reminder! updated it. Wasn't exactly sure what values they should be, so I figured we might as well make them as high as we can. Happy to change that if there's a better way to decide.

**Review by christine-sfg — APPROVED** [review]

**Pull request merged**